dex/src/main/resources/application.conf

```diff
@@ -777,6 +777,11 @@ scala.concurrent.context {
   maxExtraThreads = "256"
 }
 
+kanela {
+    show-banner = false
+    log-level = "WARNING"
```

Comment:
why do we need to set log level here?

Reply:
i guess we needn't INFO logs

Reply:
https://github.com/kamon-io/kanela/search?q=Logger.info I see nothing useful too
But WARNING is the default logging level: https://github.com/kamon-io/kanela/blob/master/agent/src/main/resources/reference.conf
Let's remove it.